src/Rules/PdoStatementExecuteMethodRule.php

```diff
@@ -71,6 +71,16 @@ private function checkErrors(MethodReflection $methodReflection, MethodCall $met
         $args = $methodCall->getArgs();
         if (\count($args) < 1) {
             $parameters = [];
+
+            $calls = $stmtReflection->findPrepareBindCalls($methodCall);
+
+            foreach ($calls as $bindCall) {
+                $args = $bindCall->getArgs();
+                $parameterName = $args[0]->value->value;
```

Comment:
this needs a `if count ($args) >= 2)` check before accessing `$args[0]` and `$args[1]`